data/data.json

```diff
@@ -1 +1,69 @@
-{"name": "Milena"}
+{
+  "mainNav": {
+    "logo": {
+      "img": "McDonals",
```

Comment:
This is not a valid url. How are you currently using this value?

Reply:
I am using this value like this: `src=mainNav.logo.link`

Reply:
This comment is about like 4, not 5.